Eastern-Research-Group/mywaterway
# Feature/381 cyan pixels to percentage

## Related Issues:
* [HMW-381](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-381)

## Main Changes:
* Converted the y-axis values of both CyAN charts to percentages, rather than pixel counts.
* Added the calculated square kilometers of pixel area to each data point.
* Enlarged chart text.
* Updated various descriptors.
* Moved the date selection slider into a shaded box.
* Combined the `StackedBarChart` and `Histogram` components into a `ColumnChart` component.
* Added zoom instructions to the histogram tooltip.

## Steps To Test:
1. Navigate to [Lake Okeechobee](http://localhost:3000/community/030902010200/monitoring), then expand the _Lake Okeechobee_ accordion item in the **Current Water Conditions** sub-tab.
2. Check the "Total Image Pixel Area" value displayed next to the y-axis of either chart, and confirm that it is slightly greater than the "Waterbody Area" value (Divide pixel square miles by .386102 to get square kilometers).
3. Hover over a data point in the stacked bar chart, take note of a percentage value, then use the percentage and total pixel area to calculate the pixel area of the data point (i.e. 517 sq mi * 1.175% = 517 * 0.01175 = 6.08 sq mi). Confirm that this matches the value next to the corresponding percentage in the tooltip.
4. Confirm that the other edits proposed by the HMW team are addressed (see the attached document in the Jira ticket).

## TODO:
- [ ] Address other considerations mentioned: if the https://cfpub.epa.gov/roe/indicator.cfm?i=95#1 link is added, how should it be labelled?
